distribution/docs/src/main/resources/_contents/_metrics/source-metrics-contents.adoc

```diff
@@ -0,0 +1,53 @@
+
+==== Source Metrics
+
+Metrics are also collected on a per source basis for each configured <<_federated_sources,Federated Source>> and <<_catalog_providers,Catalog Provider>>.
+When the source is configured, the metrics listed in the table below are automatically created.
+With each request that is either an enterprise query or a query that lists the source(s) to query these metrics are collected.
```

Comment:
This entire line just reads poorly. I get lost on the qualifying `that is either` part and can't tell when that ends and the actual statement being made resumes. 
```
With each request (that is either ... ), stuff happens to that request. 
```
In the above phrase, where is the `stuff happens` part? I get lost looking for it.